### PR TITLE
Bug 1796468: *: Complete new configuration scheme

### PR DESCRIPTION
**Pull request by brancz**
Up until this patch the whitelist was synced from the telemeter
repository being the source of truth, now the configmap in the
`/manifests` directory is and to add telemetry just this configmap is
modified.

@openshift/openshift-team-monitoring

### CODE REVIEW

File: Documentation/data-collection.md [hunk 1]
```diff
@@ -9,55 +9,55 @@ For the OpenShift 4 Developer Preview we will be sending back these exact attrib
 apiVersion: v1
 data:
   metrics.yaml: |-
-    "matches":
-    - "{__name__=~\"cluster:usage:.*\"}"
```
Comment (LiliC) [inline Documentation/data-collection.md]: Do you mind explaining why these changes?

Comment (brancz) [reply Documentation/data-collection.md]: they are just readability, I removed the surrounding quotes and the unnecessary escaping .. that happened automatically previously because of how jsonnet marshals yaml as text

**Comment by s-urbaniak** [comment]
/hold
let's verify @LiliC 's comments first

**Comment by LiliC** [comment]
@brancz we verified this, if yes we can remove the hold then.

**Comment by brancz** [comment]
/hold cancel 

The e2e tests passed with the origin telemetry test being on again. Can you lgtm this again?

**Comment by LiliC** [comment]
@brancz your generate tests are failing :O

**Comment by s-urbaniak** [comment]
the generate failures looks very unrelated though :thinking: 
```
jb: error: failed to install packages: downloading: failed to create tmp dir: stat vendor/.tmp: no such file or directory
```
@paulfantom ideas?

**Comment by paulfantom** [comment]
Yes, this needs to be reverted: https://github.com/openshift/cluster-monitoring-operator/commit/dba1836fa2bf5bb2308aa4db14ac31da7b2e246b

**Comment by s-urbaniak** [comment]
This was indeed caused by the faulty `jb` from `master` branch, A revert to the lock file output of jb 0.2.0 is available at https://github.com/openshift/cluster-monitoring-operator/pull/638.

**Comment by LiliC** [comment]
@brancz can you rebase 
/hold

as generate keeps failing
